datahub-frontend/conf/application.conf

```diff
@@ -137,6 +137,10 @@ auth.jaas.enabled = ${?AUTH_JAAS_ENABLED}
 # auth.jaas.enabled = false
 # auth.oidc.enabled = false # (or simply omit oidc configurations)
 
+# Login session expiration time
+auth.session.ttlInHours = ${?AUTH_SESSION_TTL_HOURS}
```

Comment:
Should we keep this commented out by default, like the config immediately above?

Reply:
Fixed.